soweego/importer/models/base_entity.py

```diff
@@ -35,3 +35,22 @@ class BaseEntity():
 
     def __repr__(self) -> str:
         return "<BaseEntity(catalog_id='{0}', name='{1}')>".format(self.catalog_id, self.name)
+
+
+class BaseRelationship():
+    __table_args__ = (
+        UniqueConstraint("catalog_id0", "catalog_id1"),
+    )
+
+    internal_id = Column(Integer, unique=True,
+                         primary_key=True, autoincrement=True)
+
+    catalog_id0 = Column(String(50), nullable=False, index=False)
```

Comment:
Why did you set `index=False`? Shouldn't these identifiers be indexed for faster query?

Reply:
Indexing a single column adds the constraint of uniqueness to it. This means that by indexing we cannot have multiple relations involving an entity.

Anyway looking into multicolumn indexes or something similar could be useful to improve the performance. How are we going to query the relation? Is it a classic inner join, isn't it?

Reply:
Not even a join, I would say. We just need the answer to a question like:
Which bands _Joey Ramone_ is member of?

Assuming MusicBrainz contains _Joey Ramone_ with ID `666`, the corresponding query for MusicBrainz would be something like:
```sql
SELECT to_catalog_id
FROM musicbrainz_artist_band_relationship
WHERE from_catalog_id = 666
```
then we would just leverage the links between those IDs and Wikidata QIDs.

Reply:
Makes sense. Probably a multiple column index with the two catalog ids or just a composite primary key is enough.